package.json

```diff
@@ -8,16 +8,16 @@
     "surge": "^0.18.0"
   },
   "scripts": {
-    "postinstall": "lerna clean --yes && lerna exec -- yarn --pure-lockfile --no-progress --mutex network && ./copylibs.js",
-    "update-versions": "./version.js && lerna exec -- yarn && ./copylibs.js",
-    "build": "lerna exec --ignore=generator-talend -- yarn run prepublish && ./copylibs.js",
-    "build-cmf": "lerna exec --scope=*cmf -- yarn run prepublish && ./copylibs.js --scope=cmf",
-    "build-components": "lerna exec --scope=*components -- yarn run prepublish && ./copylibs.js --scope=components",
-    "build-containers": "lerna exec --scope=*containers -- yarn run prepublish && ./copylibs.js --scope=containers",
-    "build-forms": "lerna exec --scope=*forms -- yarn run prepublish && ./copylibs.js --scope=forms",
-    "build-icons": "lerna exec --scope=*icons -- yarn run prepublish && ./copylibs.js --scope=icons && lerna exec --scope=*components -- yarn test -- -u",
+    "postinstall": "lerna clean --yes && lerna exec -- yarn --pure-lockfile --no-progress --mutex network && node copylibs.js",
```

Comment:
i know you guys don't using windows, but lets keep it cross-platform ?

Reply:
for sure !